pkgs/tools/misc/coreutils/default.nix

```diff
@@ -84,9 +85,9 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "info" ];
   separateDebugInfo = true;
 
-  nativeBuildInputs = [
-    # autoreconfHook is due to patch, normally only needed for cygwin
-    autoreconfHook
+  # autoreconfHook is due to patch, normally only needed for cygwin
+  nativeBuildInputs = optionals withAutoreconf [ autoreconfHook ]
+  ++ [
```

Comment:
It would be easier to just keep this on all platforms

Reply:
See above.